### PR TITLE
Endpoint único de registro de incidencias

### PR DESCRIPTION
**Pull request by CamiloRamirezR**
1.  Uso interno.
2.  La lógica de asignación de agente se hace en [microservicio de registro por app](https://github.com/equipo-capibaras/service-registroapp/blob/PF-221-BE-Registro-Incidente-Web/blueprints/incident.py).

## Summary by CodeRabbit

- **New Features**
	- Introduced a new incident registration API endpoint for managing incidents.
	- Added a blueprint for incident management, enhancing application modularity.
	- Implemented structured error handling for incident registration.

- **Bug Fixes**
	- Enhanced error responses for invalid input and missing fields.

- **Tests**
	- Added comprehensive unit tests for the incident registration functionality, covering various scenarios and edge cases.

### CODE REVIEW

File: blueprints/incident.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+from dataclasses import dataclass, field
+from datetime import UTC, datetime
```
[the remark this answers is not in the document]
Comment (CamiloRamirezR) [reply blueprints/incident.py]: Solved in commit [a6109d4](https://github.com/equipo-capibaras/service-incidentmodify/pull/11/commits/a6109d4354bdd1761a283e741e828f510237a4d5)

**Review by tpambor — APPROVED** [review]
